### DISCUSSION TITLE
How do i check is javascript enabled in browser?

### DISCUSSION THREAD

**Discussion by ibrhm-mnf-fingent** [vercel/next.js#16120] [Help]
My scenario is to expand all dropdowns in the bootstrap navbar (https://react-bootstrap.github.io/components/navbar/) if javascript is disabled in the browser

Is there any way to detect is javascript enabled or not in server side (inside getInitialProps)

**Comment by mismosmi** [comment]
This has been a while but you could use
```js
if (typeof window !== "undefined") {
  window.cookie = "jsEnabled=true"
}
```
And then read that cookie on the server side to know if js is enabled.
